Orange/widgets/data/owcontinuize.py

```diff
@@ -59,33 +59,34 @@ class OWContinuize(widget.OWWidget):
 
     def __init__(self):
         super().__init__()
+        my = self.my  # type: OWContinuize
```

Comment:
Doesn't work. `my.class_treatment` must resolve to `str`. `OWContinuize.class_treatment` is `bool`, and `gui` can't use it. It would work if `OWContinuize.class_treatment` were still a `Setting` and if `gui` would accept settings (besides strings, for backward compatibility).

(((On the other hand, `gui` can take the `id` of the object and parse `master.__dict__` to discover the name of the setting. Then even `self.class_treatment` would work, but this would be black magic. It's attractive, though. :)))